great_ape_safe/ape_api/rari.py

```diff
@@ -16,13 +16,19 @@ def ftoken_is_paused(self, ftoken_addr):
         return self.unitroller.borrowGuardianPaused(ftoken_addr)
 
 
-    def ftoken_pause(self, ftoken_addr):
+    def ftoken_pause(self, ftoken_addr, rf=None):
         self.unitroller._setBorrowPaused(ftoken_addr, True)
+        if rf:
```

Comment:
perhaps here is better to check if it is diff than 'None' as passing zero, will not exec this conditional block

Reply:
very nice catch